examples/terra-contract/tools/README.md

```diff
@@ -0,0 +1,113 @@
+# Tools
+This javascript package contains tools for querying and deploying the example contract.
+
+If this is the first time running the code, run the below command to install required packages: 
+
+```
+npm ci
```

Comment:
isn't this usually `npm install`? or is this some terra-specific pattern here?

Reply:
Yes i rename it.